examples/bokeh/bokeh_plot_kde_2d.py

```diff
@@ -1,6 +1,6 @@
 """
-2d KDE Bokeh
-============
+2d KDE
```

Comment:
```suggestion
2d KDE (default style)
```

See also next comment. We have a sidebar that helps navigate the examples: https://arviz-devs.github.io/arviz/examples/index.html. Now with the same title it's a bit confusing. Same should be done for matplotlib titles to match the ones here.

Reply:
I've made this change in 3b56be1